Added new kb article using-filters-in-fiddler-everywhere-for-advanced-search (#431)
telerik/fiddler-everywhere-docs: knowledge-base/using-filters-in-fiddler-everywhere-for-advanced-search.md

---
title: Using Filters in Fiddler Everywhere for Advanced Search
description: Learn how to use filters in Fiddler Everywhere to search for specific values within request and response bodies and headers.
type: how-to
page_title: Using Filters in Fiddler Everywhere for Advanced Search
slug: using-filters-in-fiddler-everywhere-for-advanced-search
tags: fiddler-everywhere, search, filters, request body, response body, headers
res_type: kb
---

## Environment

|----------|-------|
| Product  | Fiddler Everywhere |

> | Version  | 5.0.0 and above] |

Reviewer: ```suggestion
| Version  | 5.0.0 and above |
```

## Description

> Fiddler Everywhere's global search option is designed to search only within the HTTP headers of captured traffic. This is because the application is optimized for performance and user experience. Searching through the potentially large  HTTP body content in each captured session would be inefficient and slow.

Reviewer 2: ```suggestion
Fiddler Everywhere's global search option is designed to search only within the HTTP headers of captured traffic. This is because the application is optimized for performance and user experience. Searching through the potentially large HTTP body content in each captured session would be inefficient and slow.
```

However, you can use the filters option in Fiddler Everywhere to create your own filter and search for specific values within the request body, response body, and headers. Filters allow you to quickly narrow down your search results and find the desired information.

## Solution

To use filters for advanced search in Fiddler Everywhere, follow these steps:

1. Open Fiddler Everywhere.
1. Open the **Traffic** Pane.
1. In the toolbar, click on the "Filters" button.
1. In the filter panel, specify the match conditions for the request body, response body, and headers. For example, you can set the filter to match a specific value in both the request and response bodies.
1. Save the filter by giving it a name and clicking the "Save" button (top-right corner).

> 1. Apply the filter by clicking the "Apply" button (top-right corner).

Reviewer 2: ```suggestion
1. Apply the filter by clicking the "Apply" button (bottom-right corner).
```
Sorry, I had a mistake in the previous proposal.

> 1. Apply the filter by clicking the checkbox next to its name in the filter panel.

Reviewer 2: ```suggestion
1. Apply the filter by clicking the "Apply" button (top-right corner).
```
